File: pay/src/main/java/com/truenine/component/pay/api/model/response/QueryOrderApiResponseResult.java
package com.truenine.component.pay.api.model.response;

import com.wechat.pay.java.service.payments.model.Transaction;
import lombok.Data;

import java.math.BigDecimal;

@Data
public class QueryOrderApiResponseResult {

  private String orderId;
  private String orderNo;

  private BigDecimal money;
  private String tradeStatus;

}

File: pay/src/main/kotlin/net/yan100/compose/pay/models/response/QueryOrderApiResponseResult.kt
package net.yan100.compose.pay.models.response

import java.math.BigDecimal


open class QueryOrderApiResponseResult {
  open var orderId: String? = null
  open var orderNo: String? = null
  open var money: BigDecimal? = null
  open var tradeStatus: String? = null
}
